packages/web-components/src/link/link.template.ts

```diff
@@ -0,0 +1,24 @@
+import { ElementViewTemplate, html, ViewTemplate } from '@microsoft/fast-element';
+import type { Link } from './link.js';
+
+/**
+ * The template for the Button component.
```

Comment:
Should this say "Link" instead of "Button?"

Reply:
Yes - I'll push a PR 🗡️